### PR TITLE
About us

### PR DESCRIPTION
**Pull request by Z-Ramezani**
check the About us page 
figure out how to fix the tag section which I commented....

### CODE REVIEW

**Review by Madanian — CHANGES_REQUESTED** [review]
I review this. check my comments

File: front/se_project/src/Pages/admin.jsx [hunk 1]
```diff
@@ -15,19 +17,58 @@ class Admin extends Component {
...
+// export default Admin;*/
+const Admin = (props) => {
+    return ( 
+        <div>
+            <Navbar/>
+            <Breadcrumbb is1OrNot='true' first='حساب کاربری'/>
+            <div className='d-flex flex-column min-vh-100'>
+                <HorizontalLine value='آگهی‌های جدید'/>
+                <div className='container w-75 p-3 shadow bg-white rounded mb-5'>
+                        <ShowArtworkAdCost />
```
[the remark this answers is not in the document]
Comment (Shafizadegan) [reply front/se_project/src/Pages/admin.jsx:60]: why does it change?

File: front/se_project/src/Pages/artistSignUp.jsx [hunk 2]
```diff
@@ -141,8 +278,7 @@ class ArtistSignUp extends Component {
             <Footer/>
             </div>     
         </div> 
-    );
-  }
+     );
 }
  
-export default ArtistSignUp;
+export default ArtistSignUp;
```
[the remark this answers is not in the document]
Comment (Shafizadegan) [reply front/se_project/src/Pages/artistSignUp.jsx:284]: why does it change?

File: front/se_project/src/Pages/login.jsx [hunk 3]
```diff
@@ -1,51 +1,110 @@
...
             </div>
             <Footer/>
-        </div>                    
-    );
-  }
+        </div>
+     );
 }
  
 export default Login;
```
[the remark this answers is not in the document]
Comment (Shafizadegan) [reply front/se_project/src/Pages/login.jsx:110]: I don not underestand changing

File: front/se_project/src/components/ArtworkAdInfo/showArtworkAdCost.jsx [hunk 4]
```diff
@@ -4,73 +4,73 @@ import b from "../../assets/11.jpg";
...
+                        <h2 class="mt-0" style={{fontSize:'1.1vw', fontWeight:'bold'}}>توضیحات تکمیلی: {/*recive from back*/}</h2>
                     </div>
                 </div>
-            );
-        }
+            </div>
+        </div>
+    );
 }
+
```
Comment (Shafizadegan) [inline front/se_project/src/components/ArtworkAdInfo/showArtworkAdCost.jsx:74]: why does this page change?

File: front/se_project/src/components/Footer/footer.jsx [hunk 5]
```diff
@@ -2,53 +2,50 @@ import { Component } from "react";
...
         </div>
-        )
-    }
+      </MDBFooter>
+    </div>
+    </div>
+  );
 }
-
+
```
Comment (Shafizadegan) [inline front/se_project/src/components/Footer/footer.jsx:50]: علت تغییر این صفحه چی بوده؟

File: front/se_project/src/index.js [hunk 6]
```diff
@@ -1,30 +1,10 @@
...
-import Admin from './Pages/admin'; 
-import ArtworkAd from './Pages/artwokAd';
-import ViewArtistProfile from './Pages/viewArtistProfile';
-import ArtworkAdMainPage from './components/artworkAdMainPage';
-import MainPage from './Pages/mainPage';
-import UserInformation from './components/userInformation';
-import ExportProfile from './Pages/exportProfile';
+
 const root= ReactDOM.createRoot(document.getElementById('root'));
 root.render(<App/>);
```
[the remark this answers is not in the document]
Comment (Shafizadegan) [reply front/se_project/src/index.js:10]: why?

File: front/se_project/src/Pages/admin.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/Pages/admin.jsx:60]: do this after merge.

File: front/se_project/src/Pages/aboutUs.jsx [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+            <Navbar/>
+            <Breadcrumbb is1OrNot='true' first='درباره ما' />
+            <div className='container w-75 p-3 mt-4'>
+                <div className='d-flex flex-column min-vh-100'>
+                <h5><strong  style={{fontSize:'120%',color:'#C70039'}}>جامعه جهانی هنرمندان و نقاشان بزرگترین دارایی ماست.</strong></h5>
+                <h6><strong className='me-2' style={{fontSize:'90%',color:'#CC3737'}}>وقتی به بوم می‌پیوندید، به بازار هنر می‌پیوندید – دنیایی برای هنرمندان مستقل.</strong></h6>
+                <br/>
+                <h6 className='me-2' style={{fontSize:'90%'}}>ما، گروهی از دانشجویان دانشگاه اصفهان دور هم جمع شده‌ایم تا واسطه شویم میان شما علاقمندان و هنرمندان.</h6>
+                {/* https://mobirise.com/bootstrap-template/about-us-page-template/
+                هرکاری میکنم خروجی ندارم  :| */}
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/Pages/aboutUs.jsx]: fix it :)

File: front/se_project/src/App.js [hunk 8]
```diff
@@ -64,16 +67,14 @@ class App extends Component {
               <Route path='/mainPage' element={<MainPage/>}/>
               <Route path='/TicketPart' element={<TicketPart/>}/>
               <Route path='/viewArtistProfile' element={<ViewArtistProfile/>}/>
+              <Route path='/exportProfile' element={<ExportProfile/>}/>
+              <Route path='/viewExpertProfile' element={<ViewExpertProfile/>}/>
+              <Route path='/AboutUs' element={<AboutUs/>}/>
+
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/App.js:73]: done!

File: front/se_project/src/Pages/aboutUs.jsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+import AboutUsMember from '../components/aboutUsMember';
+const AboutUs = (props) => {
+    return ( 
+        <div>
+            <Navbar/>
+            <Breadcrumbb is1OrNot='true' first='درباره ما' />
+            <div className='container w-75 p-3 mt-4'>
+                <div className='d-flex flex-column min-vh-100'>
+                    <h5><strong  style={{fontSize:'2vw',color:'#C70039'}}>جامعه جهانی هنرمندان و نقاشان بزرگترین دارایی ماست.</strong></h5>
+                    <h6><strong className='me-2' style={{fontSize:'1.1vw',color:'#CC3737'}}>وقتی به بوم می‌پیوندید، به بازار هنر می‌پیوندید – دنیایی برای هنرمندان مستقل.</strong></h6>
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/Pages/aboutUs.jsx:18]: gets better..

File: front/se_project/src/components/header-slide.jsx [hunk 10]
```diff
@@ -2,38 +2,35 @@ import { Component } from "react";
...
+            <button className='carousel-control-next' type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide="next">
+                <span className='carousel-control-next-icon' aria-hidden="true"></span>
+                <span className='visually-hidden'>Next</span>
+            </button>
+        </div>
+     );
 }
-
+ 
 export default HeaderSlide;
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/components/header-slide.jsx:36]: done!

File: front/se_project/src/components/horizontalLine/horizontalLine.jsx [hunk 11]
```diff
@@ -1,16 +1,11 @@
...
+    return ( 
+        <div className="text-hr text-hr--right">
+                <span className="text-hr__text" style={{fontSize:'1.1vw'}}>{props.value}</span> 
             </div> 
-        )
-    }
+     );
 }
-
 export default HorizontalLine;
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/components/horizontalLine/horizontalLine.jsx:11]: done!

File: front/se_project/src/components/navBar/navLink.jsx [hunk 12]
```diff
@@ -1,18 +1,32 @@
...
+            <li className="nav-item">
+                <Link className="nav-link" aria-current={props.ariacurrent} to="/" style={{fontSize:'1.1vw'}}>
+                        {props.text}
+                </Link>
+            </li>
+        </div>
+     );
 }
  
 export default NavLink;
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/components/navBar/navLink.jsx:32]: done!

File: front/se_project/src/components/userInformation.jsx [hunk 13]
```diff
@@ -1,22 +1,138 @@
...
+        axios
+            .get('http://localhost:8000/api/showNumber')
+            .then((res) => {
+                // console.log(res);
+                setShowNumber(res.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/components/userInformation.jsx:132]: its ms.shafizadegan codes, not mine.

File: front/se_project/src/index.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/index.js:10]: because we don't need them.

**Review by aliinarimanii — COMMENTED** [review]
بررسی شد.

**Pull request merged**